inc/cleantalk-ajax.php

```diff
@@ -386,7 +386,9 @@ function ct_ajax_hook($message_obj = null)
         'pm_check_user_exist',
         //Cartbounty plugin (saves every action on the page to keep abandoned carts)
         'cartbounty_pro_save', 'cartbounty_save',
-        'wpmtst_form2' //has direct integration StrongTestimonials
+        'wpmtst_form2', //has direct integration StrongTestimonials
+        'rednao_smart_forms_save_form_values', //has direct integration
```

Comment:
Use apbct_is_skip_request() instead of placing these strings here.

Reply:
Done